CI: `tfproviderdocs` task failing

For example https://github.com/hashicorp/terraform-provider-aws/actions/runs/11054981668/job/30713380896:

```
Run terraform -v
  terraform -v
  make tfproviderdocs
  shell: /usr/bin/bash -e {0}
  env:
    AWS_DEFAULT_REGION: us-west-2
    TERRAFORM_VERSION: 1.8.3
/home/runner/work/_temp/bd67180d-39d3-428e-bec1-d1eda8ff37f7.sh: line 1: terraform: command not found
Error: Process completed with exit code 127.
```

--- a/.github/workflows/provider.yml
+++ b/.github/workflows/provider.yml
@@ -281,6 +281,10 @@ jobs:
       - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
         with:
           go-version-file: go.mod
+      - uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd
+        with:
+          terraform_version: ${{ env.TERRAFORM_VERSION }}
+          terraform_wrapper: false
       - uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
         continue-on-error: true
         timeout-minutes: 2

--- a/.github/workflows/resource-counts.yml
+++ b/.github/workflows/resource-counts.yml
@@ -19,6 +19,10 @@ jobs:
           installation_retrieval_payload: ${{ secrets.INSTALLATION_ID }}
           private_key: ${{secrets.APP_PEM }}
       - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+      - uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd
+        with:
+          terraform_version: ${{ env.TERRAFORM_VERSION }}
+          terraform_wrapper: false
       - run: |
           touch main.tf
           cat << EOF > main.tf

